src/System.Numerics/ref/Compat/System.Numerics.cs

```diff
@@ -27,7 +27,7 @@ public partial struct BigInteger : System.IComparable, System.IComparable<System
         public static int Compare(System.Numerics.BigInteger left, System.Numerics.BigInteger right) { return default(int); }
         public int CompareTo(long other) { return default(int); }
         public int CompareTo(System.Numerics.BigInteger other) { return default(int); }
-        public int CompareTo(object obj) { return default(int); }
+        int System.IComparable.CompareTo(object obj) { return default(int); }
```

Comment:
Basically originally we were not explicitly implementing the interface method, but in Core we are, so I moved it to match that.

---
In reply to: [70125705](https://github.com/dotnet/corefx/pull/9914#discussion_r70125705) [](ancestors = 70125705)